packages/web-components/src/components/filter-panel/filter-panel-composite.ts

```diff
@@ -118,6 +116,19 @@ class DDSFilterPanelComposite extends HostListenerMixin(StableSelectorMixin(LitE
     this.renderStatus();
   };
 
+  @HostListener('document:filterGroupViewAll')
```

Comment:
This would need to be updated with
```suggestion
  @HostListener('document:filterGroupViewAllToggle')
```

depending on the selector getter name you decide on